llmcam/file_manager.py

```diff
@@ -15,3 +15,13 @@ def list_image_files()->str:
     file naming scheme is "cap_%Y.%m.%d_%H:%M:%S_<place name>.jpg"
     """
     return "\n".join(glob.glob(os.getenv("LLMCAM_DATA", "../data")+"/"+"cap_*.jpg"))
+
+# %% ../nbs/13_fle_manager.ipynb 8
+def list_detection_files() -> str:
+  """List all detection images. File name starts with detection_"""
+  return "\n".join(glob.glob(os.getenv("LLMCAM_DATA", "../data")+ "/" + "detection_*.jpg"))
+
+# %% ../nbs/13_fle_manager.ipynb 9
+def list_demo3_files() -> str:
+  """List all parking lot frames. File name starts with demo3_"""
+  return "\n".join(glob.glob(os.getenv("LLMCAM_DATA", "../data")+ "/" + "demo3_*.jpg"))
```

Comment:
Can we avoid adding this new function but using the original function with some "prompt" information?